src/coreclr/nativeaot/System.Private.CoreLib/src/CompatibilitySuppressions.xml

```diff
@@ -956,6 +956,38 @@
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Threading.LockHolder</Target>
   </Suppression>
+  <Suppression>
```

Comment:
These can be revered. Reflection blocking was deleted from the product in #85810.